### DISCUSSION TITLE
3d cascade

### DISCUSSION THREAD

**Discussion by KhalilKilani** [MIC-DKFZ/nnUNet#1386] [Q&A]
Hi,

I attempted to run a neural network UNet cascade configuration, but I encountered an issue while trying to execute the 3d_lowres component. The problem arose from the fact that the planning process for this specific configuration did not function properly. I suspect that I may have made an error in the "nnUNetv2_plan_and_preprocess -d DATASET_ID --verify_dataset_integrity" command, which is preventing me from planning for the 3d_lowres and 3d_cascade_fullres components. The default settings for the command only include planning for 2D and 3D_fullres, so I am unsure how to modify the command to include the additional components I require.

A full line of command to use for this purpose would be verry helpful!

Thanks.

**Comment by FabianIsensee** [comment]
https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/how_to_use_nnunet.md

> Note that not all U-Net configurations are created for all datasets. In datasets with small image sizes, the U-Net cascade (and with it the 3d_lowres configuration) is omitted because the patch size of the full resolution U-Net already covers a large part of the input images.
